src/utils.js

```diff
@@ -73,10 +72,18 @@ const utils = {
 		return typeof fn === "function";
 	},
 
+	isString(s) {
+		return typeof s === "string";
+	},
+
 	isObject(o) {
 		return o !== null && typeof o === "object";
 	},
 
+	isPlainObject(o) {
+		return o !=null ? Object.getPrototypeOf(o) === Object.prototype || Object.getPrototypeOf(o) === null : false;
+	},
```

Comment:
Please add unit tests for them.

Reply:
Ops... I forget that.

Reply:
Done... 

Doing the tests a observed the isObject test was incomplete, then i added more tests. Doing that i observed that object-wrapped strings was not handled and i added a fix to that;

Reply:
Great, thanks!